revenue_app/views.py

```diff
@@ -178,8 +174,8 @@ def get_context_data(self, *args, **kwargs):
             **self.request.GET.dict(),
         )[ORGANIZER_COLUMNS]
         context['transactions'] = organizer_transactions
-        context['organizer_details'] = organizer_details(eventholder_user_id)
-        context['organizer_total'] = summarize_dataframe(organizer_transactions)
+        # TODO: Will be implemented in the next commit
```

Comment:
It would be nice to add the ticket to do this if it exists

Reply:
This was added just now.